Rakefile

```diff
@@ -0,0 +1,3 @@
+require 'simp/rake/pupmod/helpers'
+
```

Comment:
Some of your static assets are out of date - you may want to check them against another module.